packages/terra-button-group/src/terra-dev-site/doc/example/ButtonGroupIsBlock.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import ButtonGroup from 'terra-button-group/lib/ButtonGroup';
+
+class ButtonGroupIsBlock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedKeys: [] };
+    this.handleSelection = this.handleSelection.bind(this);
+  }
+
+  handleSelection(event, key) {
+    event.preventDefault();
+    this.setState(prevState => ({ selectedKeys: ButtonGroup.Utils.handleMultiSelectedKeys(prevState.selectedKeys, key) }));
+  }
+
+  render() {
+    return (
+      <ButtonGroup
+        id="button-block-group"
+        isBlock
+        onChange={this.handleSelection}
+        selectedKeys={this.state.selectedKey}
```

Comment:
Selection logic not currently working since you have a typo (this.state.selectedKey -> this.state.selectedKeys):
```suggestion
        selectedKeys={this.state.selectedKeys}
```

Reply:
Probably fixed!